test/integration/test/stream.js

```diff
@@ -0,0 +1,73 @@
+const {AudioConfig} = require('../../../dist/src/models/streaming/AudioConfig');
+const {RevAiStreamingClient} = require('../../../dist/src/streaming-client');
+const configHelper = require('../src/config-helper');
+const fs = require('fs');
+const assert = require('assert');
+
+(async (done) => {
+    const audioConfig = new AudioConfig("audio/x-raw", 'interleaved', 16000, 'S16LE', 1);
+    const client = new RevAiStreamingClient(configHelper.getApiKey(), audioConfig);
+    client.baseUrl = `wss://${configHelper.getBaseUrl()}/speechtotext/v1alpha/stream`;
+        
+    client.on('close', (code, reason) => {
+        console.log(`Connection closed, ${code}: ${reason}`);
+        assert.equal(code, 1000, `Expected close code to be [1000] but was ${code}`);
```

Comment:
Can we move these to another helper like the other asserts? Would be nice to have all the assertions at the bottom of the file in an easy to find place